src/components/shared/input/index.tsx

```diff
@@ -55,12 +69,15 @@ const Input = <
             value={value}
             onFocus={onFocus}
             w={{
-              base: '75%',
-              md: '25%',
+              base: w?.base,
+              md: w?.md,
```

Comment:
is not necessary to pass `base` and `md` attributes, as we are not developing a responsive application with breakpoints

Reply:
Done.